projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-card-hero-img',
+  templateUrl: './card-hero-img.component.html',
+  styleUrls: [ './card-hero-img.component.scss' ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgCardHeroImageComponent {
+  @Input() cover = false;
+  @Input() src: string;
+  @Input() alt = '';
+  @HostBinding('class') get class(): string {
+    if (this.src) {
```

Comment:
Not fussed about this but you could write it as a one line: 

```
return this.src ? 'lg-card-hero-img__img' : 'lg-card-hero-img__icon';
```

Reply:
Done :)